polls/models.py

```diff
@@ -1,5 +1,28 @@
 from __future__ import unicode_literals
+import datetime
 
 from django.db import models
+from django.utils import timezone
+from django.utils.encoding import python_2_unicode_compatible
 
-# Create your models here.
+
+@python_2_unicode_compatible
+class Question(models.Model):
+    question_text = models.CharField(max_length=200)
```

Comment:
The model's name is already `Question` so there is no point in giving `question_` at the beginning of this attribute. Simple `text` would be enough.